ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -2470,7 +2487,7 @@ public sealed class Commands {
 				// ReSharper disable once RedundantSuppressNullableWarningExpression - required for .NET Framework
 				string startingKey = key!;
 
-				using (IEnumerator<Bot> botsEnumerator = Bot.Bots.Where(bot => (bot.Value != Bot) && bot.Value.IsConnectedAndLoggedOn && bot.Value.Commands.Bot.HasAccess(steamID, BotConfig.EAccess.Operator)).OrderByDescending(bot => Bot.BotsComparer?.Compare(bot.Key, Bot.BotName) > 0).ThenBy(static bot => bot.Key, Bot.BotsComparer).Select(static bot => bot.Value).GetEnumerator()) {
+				using (IEnumerator<Bot> botsEnumerator = Bot.Bots.Where(bot => (bot.Value != Bot) && bot.Value.IsConnectedAndLoggedOn && ((access >= EAccess.Owner) || ((steamID != 0) && new SteamID(steamID).IsIndividualAccount && (bot.Value.GetAccess(steamID) >= EAccess.Operator)))).OrderByDescending(bot => Bot.BotsComparer?.Compare(bot.Key, Bot.BotName) > 0).ThenBy(static bot => bot.Key, Bot.BotsComparer).Select(static bot => bot.Value).GetEnumerator()) {
```

Comment:
`access >= EAccess.Owner` - This is either always `true` or always `false`; Compute it outside the `.Where(...)`
`(steamID != 0) && new SteamID(steamID).IsIndividualAccount` - as above

Reply:
A greater or equal check against a byte is not going to be more costly than a boolean check, and .NET should be already smart enough to inline it if possible.

Likewise with `steamID` check, but here we unnecessarily create new struct and check each loop - that can be indeed handled once.